spec/models/email_export_spec.rb

```diff
@@ -24,7 +24,7 @@
 
       it 'returns all leaders without any scoping' do
         ee = described_class.new.generate
-        expect(ee.leaders).to match(leaders)
+        expect(ee.leaders).to match_array(leaders)
```

Comment:
This will conflict with the changes in #416, and I think for these tests specifically we do want to check order (since the model uses queries with order defined).